concent_api/core/tests/test_core_views.py

```diff
@@ -286,6 +286,40 @@ def test_send_reject_message_save_as_receive_out_of_band_status(self):
         self.assertEqual(reject_response.status_code, 202)
         self.assertEqual(Message.objects.last().type, message.RejectReportComputedTask.TYPE)
 
+    def test_send_should_return_http_400_if_task_to_compute_deadline_is_wrong(self):
+        compute_task_def = message.ComputeTaskDef()
+        compute_task_def['task_id'] = '8'
+
+        for deadline in [-11, 'a11b', int(dateutil.parser.parse("2017-11-17 11:00:00").timestamp())]:
```

Comment:
This test is broken. `int(dateutil.parser.parse("2017-11-17 11:00:00").timestamp())` is a valid deadline but it does not fail.

You should test both valid and invalid values.